content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -86,7 +86,7 @@ enforced/disallowed:
 		<tr>
 			<td>Capabilities</td>
 			<td>
-				Adding additional capabilities beyond the <a href="https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities">default set</a> must be disallowed.<br>
+				Adding <tt>NET_RAW</tt> or capabilities beyond the <a href="https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities">default set</a> must be disallowed.<br>
```

Comment:
I don't think this change should be bundled in this PR, because it's not related to the PSP deprecation and replacement.

Let's argue about this in https://github.com/kubernetes/website/pull/27882 instead please.

Reply:
I thought we were pretty set on forbidding explicitly adding NET_RAW for baseline? https://github.com/kubernetes/website/pull/27882 is discussing whether to require dropping it for restricted, which IMO is a separate discussion.

Reply:
Since its in the default capability set of all the common runtimes, forbidding adding it feels like it makes a philosophical problem: if you keep quiet (use default capability set) you get CAP_NET_RAW access, but if you speak up (drop all, add back what you want) you can't have CAP_NET_RAW.

Reply:
We discussed this in one of the PSP replacement meetings. I can see if I can dig up the recording if you want...

The issue is that the default capability set is configurable at the runtime layer, but we're trying to avoid making the PSS configurable. If we don't forbid explicitly adding CAP_NET_RAW, then it means that you can't configure the runtime default set to drop CAP_NET_RAW without this allowing a privilege escalation. Allowing the rest of the capabilities except CAP_NET_RAW is a compromise, since dropping NET_RAW is the most frequent change to the default set.

>  Unless there's a movement under way to modify the runtimes' default capability set to remove CAP_NET_RAW... In that case, forbidding adding CAP_NET_RAW under baseline feels like a way to support the success of that movement. But if so, maybe we should do it after the runtimes remove it from their default set.

I don't know how much momentum there is for changing the "default default" set at the runtimes, but I know lots of security conscious users (e.g. openshift) are dropping it from the default set.

Reply:
I'm convinced.

It would be great to say "you can't add anything" except that the "drop all, re-add what you want" workflow is so important.

Let's go ahead and forbid "drop all, re-add cap_net_raw" even though it's philosophically discordant. It's better than the alternatives.